src/main.rs

```diff
@@ -129,21 +130,25 @@ fn main() {
 
     // check that cleaned file is smaller
     if buf.len() > raw.len() {
-        println!("Error: Cleaned file is bigger than original.");
+        writeln!(stderr(), "Error: Cleaned file is bigger than original.").unwrap();
         on_err();
         return;
     }
 
     // save buffer
-    try_msg!(cleaner::save_file(&buf[..], out_file));
+    if cli::get_flag(&args, Key::Stdout) {
+        try_msg!(cleaner::write_stdout(&buf[..]));
+    }
 
-    if cli::get_flag(&args, Key::Quiet) {
-        return;
+    if !out_file.is_empty() {
```

Comment:
Since we switched to `Option<String>` it will became:
`if let Some(out_file) = out_file {`